isign/archive.py

```diff
@@ -117,15 +117,15 @@ def precheck(self):
                 plist_bytes = z.read(plist_path)
                 plist = biplist.readPlistFromString(plist_bytes)
                 is_native = is_info_plist_native(plist)
-                log.debug("is_native: {}".format(is_native))
+                log.info("is_native: {}".format(is_native))
             if len(apps) > 1:
                 log.warning('more than one app found in archive')
 
         return (relative_app_dir, is_native)
 
     def unarchive_to_temp(self):
         containing_dir = make_temp_dir()
-        call([get_helper('unzip'), "-qu", self.path, "-d", containing_dir])
+        call([get_helper('unzip'), "-oqq", self.path, "-d", containing_dir])
```

Comment:
`-o` should be unnecessary; as far as I can tell that overwrites files, but the directory should be brand new. Why did you change this?